src/pages/jobs.js

```diff
@@ -0,0 +1,160 @@
+    document.getElementById('scroll-to').scrollIntoView({ behavior: 'smooth', block: 'start' })
+
+    let what = 'javascript Reactjs gatsby graphql nodejs node.js jquery bootstrap'
+    let exclude = '0000 senior sr principal lead master'
+
+    let url = `https://8oi4im0qo3.execute-api.us-east-1.amazonaws.com/dev/jobsearch/${page}?&results_per_page=15&what_or=${what}&where=${
+      formResponse.zipCode
+    }&distance=${formResponse.distance}&what_exclude=${exclude}&sort_by=date&max_days_old=30${
+      formResponse.remote === true ? '&what_and=remote' : ''
+    }`
+
+    fetch(url)
+      .then(response => response.json())
+      .then(data => {
+        setJobData(data)
+        setPageContext({
+          currentPage: page,
+          minPage: Math.floor(page / 10) * 10,
+          maxPage: Math.ceil(page / 10) * 10 - 1,
+          totalPages: Math.floor(data.count / 15),
+          formResponse: formResponse,
+          formData: formData,
+          setPageContext: setPageContext,
+        })
+        console.log(pageContext)
+      })
+      .then(setJobData(false))
+      .catch(console.error)
+  }
+
+  return (
+    <>
+      <SEO title="Job Search" />
+      <PageHeader />
+
+      {/* Section Header */}
+      <section id="jobs" className="small-top-pad section bg-default" style={{ paddingBottom: 35 }}>
+        <div className="container">
+          <div className="row">
+            <div className="col-md-12 lead-in">
+              <h1 className="story-title">JOB SEARCH</h1>
+              <i>
+                <p className="pt-0">
+                  #VetsWhoCode provides job placement assistance to veterans and spouses.
+                </p>
+              </i>
+            </div>
+            <div className="row">
+              <div className="col-md-12">
+                <div className="faq-short-brief container-fluid">
+                  <p>
+                    Vets Who Code Job Search (VWC) is a tool for connecting veterans, military, and
+                    military spouses with jobs. Our goal is to make every workplace fair and
+                    profitable by bringing together the perfect candidate with the ideal employer.
+                    Our site pairs technology and best practices in order to promote practical and
+                    gainful employment.
+                  </p>
+                </div>
+              </div>
+            </div>
+            {/*  End Header  */}
+
+            {/*  Search Bar  */}
+            <div id="scroll-to" className="scroll-to"></div>
+            <div className="search">
+              <div>
+                <Form formData={formData} />
+              </div>
+            </div>
+            {/*  Search End  */}
+
+            {/*  No Results  */}
+            <div className="no-results" style={{ marginTop: 75 }}>
+              <p
+                className={`text-center ${
+                  formSubmitted && jobData && jobData.results.length === 0 ? '' : 'hidden'
```

Comment:
You can use optional chaining here. `jobaData?.results.length` is the same as `jobData && jobData.results.length`

Reply:
Cool! all set.